hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -144,20 +145,19 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
       const dataSize = duResponse.size;
       let subpaths: IDUSubpath[] = duResponse.subPaths;
 
-      subpaths.sort((a, b) => (a.size < b.size) ? 1 : -1);
-
       // Only show top n blocks with the most DU,
       // other blocks are merged as a single block
-      if (subpaths.length > limit) {
+      if (subpaths.length > limit || (subpaths.length > 0 && limit === MAX_DISPLAY_LIMIT)) {
```

Comment:
@smitajoshi12 Could you explain why we need the other condition? 

`(subpaths.length > 0 && limit === MAX_DISPLAY_LIMIT)` - we have any number (not zero) of subpaths and the limit is set to 30, the max display limit. This means that even even if we don't have more subpaths than the limit, we are adding the `Other Objects`? Why is that needed?

Reply:
@dombizita 

1)subpaths.length > 0 && limit === MAX_DISPLAY_LIMIT  this condition is purely for Limit=30 then we showing other objects in Pie Chart

2) Previous Conditions are for Example out of 30 in from Dropdown we have selected to show only 10 Enties in Pie Chart Length=30 and our limit=10 then other 20 entities we are showing as other object.